scene/gui/tab_bar.h

```diff
@@ -95,6 +95,8 @@ class TabBar : public Control {
 
 	bool select_with_rmb = false;
 
+	bool input_rollover = false;
```

Comment:
```suggestion
	bool input_rollover = false;
```
Avoid adding too many blank lines.

Reply:
Fixed in this case